components/service/experiments/src/main/java/mozilla/components/service/experiments/Experiments.kt

```diff
@@ -111,6 +101,21 @@ open class ExperimentsInternalAPI internal constructor() {
         isInitialized = false
     }
 
+    internal fun getGlean(): GleanInternalAPI {
+        return Glean
+    }
+
+    internal open fun getExperimentsUpdater(context: Context): ExperimentsUpdater {
```

Comment:
If this function isn't used outside of this class I suggest adding the `@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)` annotation.